result/go/snyk.txt

```diff
@@ -0,0 +1,1114 @@
+
+Testing docker-scanning-test-go...
+
+Organization:      hogo6002
+Package manager:   npm
+Target file:       /root/go/pkg/mod/github.com/hashicorp/vault@v1.14.1/website/package.json
+Project name:      vault-docs
+Docker image:      docker-scanning-test-go
+Licenses:          enabled
+
+✔ Tested docker-scanning-test-go for known issues, no vulnerable paths found.
+
+-------------------------------------------------------
+
+Testing docker-scanning-test-go...
+
+Tested 16 dependencies for known issues, found 1 issue.
+
+
+Issues with no direct upgrade or patch:
+  ✗ Regular Expression Denial of Service (ReDoS) [Medium Severity][https://security.snyk.io/vuln/SNYK-JS-D3COLOR-1076592] in d3-color@1.4.1
+    introduced by d3-flame-graph@4.1.3 > d3-transition@3.0.1 > d3-color@1.4.1 and 2 other path(s)
+  This issue was fixed in versions: 3.1.0
+
+
+
+Organization:      hogo6002
+Package manager:   npm
+Target file:       /usr/share/go-1.21/src/cmd/vendor/github.com/google/pprof/third_party/d3flamegraph/package.json
+Project name:      d3-pprof
+Docker image:      docker-scanning-test-go
+Licenses:          enabled
+
+Snyk wasn’t able to auto detect the base image, use `--file` option to get base image remediation advice.
+Example: $ snyk container test docker-scanning-test-go --file=path/to/Dockerfile
+
+Snyk found some vulnerabilities in your image applications (Snyk searches for these vulnerabilities by default). See https://snyk.co/app-vulns for more information.
+
+To remove these messages in the future, please run `snyk config set disableSuggestions=true`
+
+-------------------------------------------------------
+
+Testing docker-scanning-test-go...
+
+✗ Medium severity vulnerability found in github.com/dvsekhvalnov/jose2go
+  Description: Uncontrolled Resource Consumption ('Resource Exhaustion')
+  Info: https://security.snyk.io/vuln/SNYK-GOLANG-GITHUBCOMDVSEKHVALNOVJOSE2GO-6137739
+  Introduced through: github.com/dvsekhvalnov/jose2go@v1.5.0
+  From: github.com/dvsekhvalnov/jose2go@v1.5.0
+  Fixed in: 1.6.0
+
+
+
+Organization:      hogo6002
+Package manager:   gomodules
+Target file:       /src/final/main
+Project name:      final
+Docker image:      docker-scanning-test-go
+Licenses:          enabled
+
+Tested 34 dependencies for known issues, found 1 issue.
+
+Snyk wasn’t able to auto detect the base image, use `--file` option to get base image remediation advice.
+Example: $ snyk container test docker-scanning-test-go --file=path/to/Dockerfile
+
+Snyk found some vulnerabilities in your image applications (Snyk searches for these vulnerabilities by default). See https://snyk.co/app-vulns for more information.
+
+To remove these messages in the future, please run `snyk config set disableSuggestions=true`
+
+-------------------------------------------------------
+
+Testing docker-scanning-test-go...
+
+✗ Medium severity vulnerability found in google.golang.org/protobuf/encoding/protojson
```

Comment:
This vuln didn't get detected by osv-scanner nor other tools. It also not on osv.dev.